shopinvader_sale_coupon/services/abstract_sale.py

```diff
@@ -39,7 +43,14 @@ def _convert_one_line(self, line):
         return res
 
     def _parser_coupon(self):
-        return ["id", "code", "state", "expiration_date", "partner_id", "program_id"]
+        return [
+            "id",
+            "code",
+            "state",
+            "expiration_date",
+            ("partner_id:partner", self.component(usage="addresses")._json_parser()),
```

Comment:
we don't need full data here IMO. Simply pass `["id", "name"]`

Reply:
Done!